problem_builder/tests/integration/xml/dashboard_1.xml

```diff
@@ -0,0 +1,58 @@
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+        <pb-mcq display_name="1.2 Second MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+        <pb-mcq display_name="1.3 Third MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+        <html_demo> This message here should be ignored. </html_demo>
+    </problem-builder>
+    <problem-builder display_name="Step 2">
+        <pb-mcq display_name="2.1 First MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="4">Option 4</pb-choice>
+            <pb-choice value="5">Option 5</pb-choice>
+            <pb-choice value="6">Option 6</pb-choice>
+        </pb-mcq>
+        <pb-mcq display_name="2.2 Second MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+        <pb-mcq display_name="2.3 Third MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+    </problem-builder>
+    <problem-builder display_name="Step 3">
+        <pb-mcq display_name="3.1 First MCQ" question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="1">Option 1</pb-choice>
+            <pb-choice value="2">Option 2</pb-choice>
+            <pb-choice value="3">Option 3</pb-choice>
+            <pb-choice value="4">Option 4</pb-choice>
+        </pb-mcq>
+        <pb-mcq display_name="3.2 MCQ with non-numeric values"
+          question="Which option?" correct_choices='["1","2","3","4"]'>
+            <pb-choice value="A">Option A</pb-choice>
+            <pb-choice value="B">Option B</pb-choice>
+            <pb-choice value="C">Option C</pb-choice>
+        </pb-mcq>
+    </problem-builder>
+    <pb-dashboard mentoring_ids='["dummy-value"]'>
+    </pb-dashboard>
```

Comment:
These three XML files seem to be identical except for the Problem Builder block at the end.

If you want, you can just use one XML file, and make this part a template and insert the `<pb-dashboard/>` part via code before setting it as the scenario. There's also a `self.load_root_xblock` method in `SeleniumXBlockTest` that allows you to modify a block's fields via code and then refresh the scenario.